composer.json

```diff
@@ -53,7 +53,7 @@
     "drupal/rabbit_hole": "^1.0@beta",
     "drupal/redirect": "^1.4",
     "drupal/styleguide": "^1",
-    "drupal/svg_image": "^1.9",
+    "drupal/svg_image": "^1",
```

Comment:
It must point to `^1.10` as it's security release

See https://www.drupal.org/project/svg_image/releases/8.x-1.10

Reply:
Updated